README.md

```diff
@@ -35,7 +35,7 @@ Syntax
 For example:
 
 ``` ocaml
-# type loc = [%import: Location.t];;
+# type%import loc = Location.t;;
 type loc = Location.t = { loc_start : Lexing.position; loc_end : Lexing.position; loc_ghost : bool; }
 # module type Hashable = [%import: (module Hashtbl.HashedType)];;
```

Comment:
As I mentioned in https://github.com/ocaml-ppx/ppx_import/pull/68#issue-1122253293, that would be nice to change the syntax of module type declaration too.

Reply:
Happy to open a follow-up PR with address that :)